### ISSUE TITLE
Weird Callout on mobile (iOS)

### ISSUE THREAD

**Issue by Abeltjes** [efemkay/obsidian-modular-css-layout#29]
Hi!

Thanks for the lovely CSS for building multi-column things in Obsidian. I genuinely 

I stumbled across some weird behavior of the multi-column callouts on mobile. I only tried the IOS app, not the android one. Sometimes the callouts do not work (I tested it with the Default theme and the Minimal theme) but sometimes they do (on the Sanctum theme - my favorite, hurray!). 

In the sanctum theme, however, something weird happens with callouts within the multi-column callouts. See this screenshot:
![IMG_5722](https://user-images.githubusercontent.com/49521042/212049142-6989197d-a057-491a-967d-12d5e75770cb.PNG)
The margins in the callouts seem off. My knowledge of css is very limited. Do you recon there is some way to fix it?

Kind regards and keep up the good work!

**Comment by efemkay** [comment]
Yeah possibly some margin special to mobile I wasn't aware of. I'll have a look at it.

**Comment by efemkay** [comment]
@Abeltjes i cannot reproduce that with Sanctum theme (nor any other theme). Can you provide the raw markdown of that screenshot. Also, is there any other snippets u enabled?
